includes/definitions/zyxelwlc.yaml

```diff
@@ -0,0 +1,20 @@
+os: zyxelwlc
+text: 'Zyxel Zyxelwlc'
+group: zyxel
+type: wireless
+over:
+    - { graph: device_bits, text: Traffic }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+    - { graph: device_sensor-count-zyxelwlc-sessions, text: 'Active Sessions' }
```

Comment:
As you identified, can you please adapt this line ?

Reply:
It' s done  and works well. The /opt/librenms/includes/html/graphs/device/sensor-count-zyxelwlc-sessionNum.inc.php is needed because it gives the grahpTemplate. Without it  GraphTEmplate missing  error message appears in the over section ....
May be I have CR/LF problem as well. The pass check is  too sensitive :-)

Reply:
It is not too elegant solution but working. I think there are problem with sensor part of yamldiscovery.php or  in includes/html/graphs/device/sensor.inc.php.
The wireless_sensors graph does not require unique graph template but sensors does if i want to display them in 'over' section. Now it is too complicated to me to rewrite the yamldiscovery and/or sensor.inc.php.
The includes/html/graphs/device/wireless-sensor.inc.php and other wireless_xxxxx.inc.php are working well.

Reply:
Yep, so far there is no way to show a single sensor in the over view. Only a family of sensors (like all count sensors). 
That PR is fine with me.